src/main/java/uk/nhs/prm/e2etests/model/request/PdsAdaptorRequest.java

```diff
@@ -1,13 +1,19 @@
 package uk.nhs.prm.e2etests.model.request;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import java.util.Objects;
 
-@Data
-@AllArgsConstructor
-@NoArgsConstructor
-public class PdsAdaptorRequest {
-    private String previousGp;
-    private String recordETag;
-}
+public record PdsAdaptorRequest(String previousGp,
+                                String recordETag) {
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This can be replaced by Lombok's default Equals implementation as it is comparing every field in the record.